datafusion/proto/src/from_proto.rs

```diff
@@ -829,10 +839,32 @@ impl TryFrom<&protobuf::ScalarValue> for ScalarValue {
                 )
             }
             Value::Date64Value(v) => Self::Date64(Some(*v)),
-            Value::TimeSecondValue(v) => Self::TimestampSecond(Some(*v), None),
-            Value::TimeMillisecondValue(v) => Self::TimestampMillisecond(Some(*v), None),
             Value::IntervalYearmonthValue(v) => Self::IntervalYearMonth(Some(*v)),
             Value::IntervalDaytimeValue(v) => Self::IntervalDayTime(Some(*v)),
+            Value::TimestampValue(v) => {
```

Comment:
the replication is unfortunate but I see it is not something that got introduced with this PR